static/app/views/issueDetails/traceTimelineOrRelatedIssue.spec.tsx

```diff
@@ -49,7 +49,7 @@ describe('TraceTimeline & TraceRelated Issue', () => {
         'project.name': project.name,
         title: 'Slow DB Query',
         id: 'abc',
-        transaction: '/api/slow/',
+        transaction: 'n/a',
```

Comment:
`transaction` is not used anymore when creating the message for issue platform events.